strapi/strapi
# Api token v2/backend expirations

### What does it do?

- adds `lifespan` and `expiresAt` to tokens
- checks `expiresAt` when authenticating token and only allow access if token has not expired
- adds tests for token verification

### Why is it needed?

To allow tokens to expire

### How to test it?

Create a token with a lifespan and ensure that it is created with an expiresAt field and that it stops working after that time.

Tests have also been provided.

### Related issue(s)/PR(s)
